javascript/big-practice/src/scripts/models/listNoteModel.js

```diff
@@ -21,17 +26,33 @@ export default class ListNoteModel {
    *
    * @returns {Object} note
    */
-  addNote(title, description) {
-    const noteItem = {
-      id: new Date().getTime().toString(),
-      title,
-      description,
-      isTrash: false,
-    };
-
-    const note = new NoteModel(noteItem);
-    this.notes.push(note);
-    LocalStorage.setItems(STORAGE_KEYS.LIST_NOTE, this.notes);
+  async addNote(title, description) {
+    let note;
+
+    if (LocalStorage.getItems(STORAGE_KEYS.USERNAME)) {
+      const noteItem = {
+        id: new Date().getTime().toString(),
+        title,
+        description,
+        isTrash: false,
+        owner: LocalStorage.getItems(STORAGE_KEYS.USERNAME),
+      };
+      note = noteItem;
+
+      await postData(noteItem);
+    }
+
+    if (!LocalStorage.getItems(STORAGE_KEYS.USERNAME)) {
+      const noteItem = {
```

Comment:
These pieces of information are the same, just different have the owner field or not. You should define a variable to contain the same pieces of information after that just need to check the condition and push the needed field into that variable.